sdkms/apps_generated.go

```diff
@@ -185,9 +185,37 @@ type AppCredential struct {
 	Secret               *string
 	Certificate          *Blob
 	TrustedCa            *TrustAnchor
-	GoogleServiceAccount *struct{}
+	GoogleServiceAccount *GoogleServiceAccount
 	SignedJwt            *AppCredentialSignedJwt
 }
+
+// Key access justification reason.
+type KeyAccessJustification string
+
+// List of supported key access justifications.
+const (
+	CustomerInitiatedSupport               KeyAccessJustification = "CUSTOMER_INITIATED_SUPPORT"
+	CustomerInitiatedAccess                KeyAccessJustification = "CUSTOMER_INITIATED_ACCESS"
+	GoogleInitiatedService                 KeyAccessJustification = "GOOGLE_INITIATED_SERVICE"
+	GoogleInitiatedReview                  KeyAccessJustification = "GOOGLE_INITIATED_REVIEW"
+	GoogleInitiatedSystemOperation         KeyAccessJustification = "GOOGLE_INITIATED_SYSTEM_OPERATION"
+	ThirdPartyDataRequest                  KeyAccessJustification = "THIRD_PARTY_DATA_REQUEST"
+	ReasonUnspecified                      KeyAccessJustification = "REASON_UNSPECIFIED"
+	ReasonNotExpected                      KeyAccessJustification = "REASON_NOT_EXPECTED"
+	ModifiedCustomerInitiatedAccess        KeyAccessJustification = "MODIFIED_CUSTOMER_INITIATED_ACCESS"
+	ModifiedGoogleInitiatedSystemOperation KeyAccessJustification = "MODIFIED_GOOGLE_INITIATED_SYSTEM_OPERATION"
+	GoogleResponseToProductionAlert        KeyAccessJustification = "GOOGLE_RESPONSE_TO_PRODUCTION_ALERT"
```

Comment:
These constants should have the type name as a prefix, e.g. `KeyAccessJustificationCustomerInitiatedSupport`